src/d3dx12_property_format_table.cpp

```diff
@@ -12,7 +12,10 @@
     #define WIN32_LEAN_AND_MEAN
 #endif
 #ifndef NOMINMAX
-    #define NOMINMAX
+    #define NOMINMAX 1
+#endif
+#ifdef __MINGW32__
```

Comment:
This doesn't need to be `#ifdef`'d, it's always safe to include. You can just put it after the `winadapter.h` include.

Reply:
It's mostly to make it clear that it's a workaround for MinGW.